pkgs/by-name/gi/gitbutler-ui/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, mkYarnPackage
+, fetchYarnDeps
+, fetchFromGitHub
+}:
+
+mkYarnPackage rec {
+  pname = "gitbutler-ui";
+  version = "0.10.10";
+
+  src = fetchFromGitHub {
+    owner = "gitbutlerapp";
+    repo = "gitbutler";
+    rev = "release/${version}";
+    hash = "sha256-DutsgMlYBITzD65Zbq1OlbmY20RogogN57flGn9x06c=";
+  };
+
+  sourceRoot = "${src.name}/gitbutler-ui";
+
+  # The package.json must use spaces instead of upstream's tabs to pass Nixpkgs
+  # CI.
+  # To generate the Yarn lockfile, run `yarn install`.
```

Comment:
Have you tried `pnpm-lock-export`?

Reply:
I couldn't find that in my searching, thanks!
It doesn't work, though. Something to do with the way [`tauri-plugin-store-api` is listed](https://github.com/gitbutlerapp/gitbutler/blob/release/0.10.11/pnpm-lock.yaml#L218) is causing issues that `yarn install` seems to avoid.